style.css

```diff
@@ -0,0 +1,635 @@
+}
+
+.hackjos-logo img {
+  width: 100%;
+  height: 100%;
+}
+
+.hackjos-logo-text {
+  text-align: center;
+  max-width: 400px;
+  padding: 20px 50px;
+  border: 1px solid rgb(199, 195, 195);
+  background-color: rgba(255, 255, 255, 0.842);
+  margin: 0 auto;
+}
+
+.past-hackjos {
+  margin-top: 50px;
+}
+
+.hackathon-text h1 {
+  font-size: 25px;
+}
+
+.hackathons {
+  justify-content: center;
+}
+
+.hackathons h3,
+p {
+  font-weight: 700;
+}
+
+.hackathon1 h3 {
+  color: #fff;
+}
+
+.hackathon1 p {
+  color: #fff;
+}
+
+.hackathon2 h3 {
+  color: #fff;
+}
+
+.hackathon2 p {
+  color: #fff;
+}
+
+.hackathon1 {
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  margin: 20px 5px;
+  padding: 50px 30px;
+  background-color: rgba(255, 68, 0, 0.507);
+}
+
+.hackathon1::before {
+  content: '';
+  width: 100%;
+  max-width: 400px;
+  position: absolute;
+  background-image: url('./images/hackjos1.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: -1;
+  opacity: 0.4;
+}
+
+.hackathon2 {
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  margin: 20px 5px;
+  padding: 50px 30px;
+  background-color: rgba(255, 68, 0, 0.507);
+}
+
+.hackathon2::before {
+  content: '';
+  width: 100%;
+  max-width: 400px;
+  position: absolute;
+  background-image: url('./images/hackjos2.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: -1;
+  opacity: 0.4;
+}
+
+.featured-speakers .section-heading {
+  color: #272a31;
+}
+
+.featured-speakers-container {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-auto-flow: row;
+  gap: 25px;
+}
+
+.all-speakers-container {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-auto-flow: row;
+  gap: 25px;
+}
+
+.featured-speakers-card {
+  position: relative;
+  max-width: 700px;
+  display: flex;
+}
+
+.featured-speakers-card::after {
+  content: '';
+  position: absolute;
+  background-image: url('./images/black-white2.jpg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 70px;
+  height: 70px;
+  top: 0;
+  left: 0;
+  opacity: 0.6;
+  z-index: -1;
+}
+
+.speaker-img {
+  width: 200px;
+  max-width: 180px;
+  max-height: 130px;
+  padding-left: 10px;
+  padding-top: 10px;
+}
+
+.speaker-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.featured-text {
+  padding-left: 20px;
+  max-width: 400px;
+}
+
+.featured-text i {
+  color: #ec5242;
+}
+
+.featured-text hr {
+  width: 30px;
+}
+
+.get-more {
+  display: none;
+  border: 1px solid #d3d3d3;
+  padding: 20px;
+  align-items: center;
+}
+
+.more {
+  margin: 0 auto;
+  border: none;
+  font-size: 20px;
+  background-color: inherit;
+}
+
+.fa-angle-down {
+  color: #ec5242;
+  padding-left: 5px;
+}
+
+.partner-section {
+  position: relative;
+  margin-top: 40px;
+  background-color: #272a31;
+  z-index: 1;
+  padding-bottom: 50px;
+}
```

Comment:
- [ ] The partner section elements should be aligned in a row in desktop view. In addition, the sizes of the logo images and spacing between each other should resemble the original design. Kindly make changes here. :pray: 

![image](https://user-images.githubusercontent.com/64894927/132694966-44276a16-5847-45cc-adb7-8139c89f6919.png)
![image](https://user-images.githubusercontent.com/64894927/132695161-bd44cbc6-7a12-423c-9740-b5641051cd56.png)